### ISSUE TITLE
The borer balance suggestions thread

### ISSUE THREAD

**Issue by despotate** [d3athrow/vgstation13#13324]
We gotta fix borers. Just gotta. I opened https://github.com/d3athrow/vgstation13/issues/13322 to try and fix chest borers and it appears to be the case that not a lot of people actually understand that different body parts are immensely unbalanced relative to each other. Let me explain by comparing head and chest borers.

Head borers, off the bat, can:

- Completely remove and prevent genetic defects, brain damage, slowness, temperature damage by spending one or two chems on the appropriate drug
- Fix toxin damage faster than anyone with antitox + charcoal
- Fix brute and burn damage fairly well with bicaridine and kelotane, only requiring a couple of chemicals to do so (e.g. 1 unit of bicaridine = 10 brute damage healed), as well as halt crit fairly easily with a squirt of inaprov

Then when they're upgraded, they can:

- Spend 2 (2!) chemicals once in a while to completely negate all suffocation damage even on someone who has 0% blood in them
- Secrete keloderm for the fastest burn damage removal
- Heal internal organ and clone damage with rezadone and peridax for essentially free after researching them. If you're keeping count, that means they can heal EVERY single damage type except for radiation damage by spending just a few chems
- Plus they have that sweet telephoto X-ray vision
- Taste blood for those precise medications

In comparison, off the bat, chest borers can:
- Get a more advanced scan than headborers, which is completely pointless because you can't fix anything anyway and anyone who could do so would be able to do their own advanced scan
- Spend a fuckton of chemicals to secrete blood. For those of you who think this is balanced, I have a question. Would you rather spend 250 borerbucks injecting 250u of blood into someone to keep them gasping and stumbling along while they continue to bleed out (unless you're gonna pay an extra 200 for the clotting agent), or be a fucking headborer, spend 250 borerbucks researching dexplus, and expend 2 chemicals every minute or so and be much more effective?
- Heal eye damage, ear damage, fatness, and drunkness. Yawn.
- Have Oxycodone which is the only thing they have that's kind of better than headborers, but not really since Tramadol is "good enough" and doesn't obscure your health data

And when upgraded, they can:
- Feed their host. YAWN
- Cure radiation damage in their host **by causing toxin damage that they can't even fix themselves, you can not make this shit up**
- Mitigate temperature damage by injecting chilis/ice chilis. remember, **headborers get leporazine for free**, meanwhile chestborers need to spend 150 borerbucks each to cure being too hot or too cold. **can NOT make this shit up**
- Secrete smoke, this is kind of fun and the only clear upgrade that a chest upgrade has over a headborer
- Secrete clotting agent after spending 200 borerbucks to research it (and it costs 10 chems per u) (recall that headborers only need spend 50 more chems for dexplus, superior in every way)
- Learn to REDUCE temperature damage in their host by spending 350 (!!!) borerbucks PER type of temperature damage, **YOU ABSOLUTELY CAN NOT MAKE THIS SHIT UP, 700 BORERBUCKS JUST TO DO (SHITTILY) WHAT A HEADBORER CAN DO FOR FREE, LITERALLY CODE THAT IS BEING RUN BY THE SERVER RIGHT AT THIS MOMENT**
- And CONSTANTLY EXPEND CHEMICALS to reduce brute and burn damage (recall: they still can't actually FIX the damage, just reduce it), that's 200 borerbucks per damage type when headborers can pump bi and keloderm like it's nothing

Oh, and does it seem like something's missing? Yes, something IS missing - CHEST BORERS CAN NOT EVEN TASTE BLOOD, THEY HAVE LITERALLY NO WAY OF FINDING OUT WHAT'S IN THEIR HOST'S BLOOD OR WHAT THEY INJECTED RECENTLY, **THERE ARE CURRENTLY 63 PEOPLE ONLINE PLAYING A GAME WHERE THIS IS ACTUALLY THE CASE, LITERALLY SOMETHING THAT YOU ARE ABLE TO BE SUBJECTED TO RIGHT NOW IF YOU PLAY AS A BORER AND CLIMB INTO SOMEONE'S CHEST INSTEAD OF THEIR HEAD (AND MAYBE 10% OF PLAYERS WILL EVER TAKE OFF THEIR SUIT TO LET YOU IN, COMPARED TO 90% OF PLAYERS KEEPING THEIR HEAD OPEN)**

tl;dr headborers are adminordrazine, chest borers are lemon juice

Also the arm and leg borers are kind of balanced already IMO, at least they're relatively balanced compared to the immense gap between chest and head borers. So when I say that chest borers should be able to defibrillate their host, you should keep in mind that I'm talking about CHEST borers, the forgotten child of borers, the cripple runt who looks at you with puppy eyes, not the megalomaniac headborer who controls all.

Share borer balance suggestions here

**Comment by jknpj** [comment]
Make chestborers do something about chestburstes because well, i'm pretty sure that ribcage won't be big enough for two parasites

**Comment by Sprok0** [comment]
You seem to have missed that chest borers can secrete clotting agent, which flat stops bleeding entirely.

**Comment by despotate** [comment]
Knew I missed something, thanks. Though clotting agent is still a clear downgrade to dexplus. Plus IIRC it costs 10 chemicals to use per unit for whatever reason.

**Comment by 9600bauds** [comment]
REMOVE XRAY remove xray

**Comment by despotate** [comment]
I don't know why everyone has a problem with X-ray and not the chems. By the time a borer researches all the important chems and then gets x-ray after telephoto, genetics would have gotten it already anyway.

**Comment by JustSumBody** [comment]
Takes way less time than the 40 or so minutes genetics takes, especially if you opt to rush vision instead of chems. 
Also consider it's GUARANTEED to apply the xray and farsight through the borer, no guarantee your SE injector will even give you xray even if you've got the right block for it.

**Comment by N3X15** [comment]
```
Probably the best thing to do would be create some slower-acting chems that do the same thing as kelotane and dex and give those to the borer
and get rid of x-ray because fucking why
possibly double cost on all chems
get rid of secrete blood and replace with iron or something[*]
Add some sort of iodine-like thing that gets rid of rads without causing tox with slower rate of heal
Add whatever that tox damage chem was

*:
despotate - Today at 5:32 PM
secrete blood is super expensive though
it's one chem per unit of blood
N3X15 - Today at 5:32 PM
doesn't matter, why would an alien slug shit human blood
despotate - Today at 5:32 PM
why would an alien slug shit anything it can shit
blood seems realistic to me idk it dupes the blood marrow or someshit?
N3X15 - Today at 5:32 PM
Because many organisms can create simple organic chems'
they can't duplicate human blood with all the right enzymes
and I mean it's even the right bloodtype
Secreting iron would be far better and more balanced
```

My 😎 opinion

**Comment by AI-Is-Rogue** [comment]
Chest borers use the advanced scanning for finding viruses, but they can't really even cure the viruses without a head to prevent the host from dying due to radium sending them to crit.

**Comment by 9600bauds** [comment]
Honestly it seems like the dermaline and bicaridine should be in the chest, not the head

**Comment by Ephx1** [comment]
Why can't borers just go back to the little slug that gave you some chems instead of what we have now, where every one has to give you some insane power?

**Comment by AI-Is-Rogue** [comment]
@Ephx1 like pre-nonantag borer chems? Because I am pretty sure head borers were just as strong as they are now if not stronger when they became non-antagonists.

**Comment by Ephx1** [comment]
No, they weren't as strong, head borers got a fuckhuge buff when this system was put into place for some godawful reason and they had nowhere near the amount of chems they do now.

I would hope I remember this right because I've done nothing but play borer and robot for the last few months.

**Comment by AI-Is-Rogue** [comment]
@Ephx1 I decided to see what they got right when they became non-antagonists, they were just about as strong, minus a few things.

**Comment by Ephx1** [comment]
And even then I felt they deserved a nerf despite constantly playing one

**Comment by AutismSpider** [comment]
move rezadone and peridax  to chest borers and most of the chems head has.

**Comment by Kurfursten** [comment]
I feel that borers are poorly balanced and stretched thin because Shadowmech felt obligated to have styles for being burrowed into different body parts. But there's three types of arm borers, leg borers, chest borers, and head borers. That's six kinds of borer, no wonder the ideas started to decay down to "it can synthesize a powergamey, difficult to get chemical" (example: latte, ham serum, and personal AI smoke are all things borers can make for some reason). I don't blame Shadowmech at all, it's super hard to come up with creative ideas and writing entirely new stuff (like the extendo-arm) is a hell of a lot harder to do than "add preexisting chemical to the chemsynth list".

I would much rather see borers boiled down into maybe 2-3 more consolidated, well rounded types (maybe not even specific to one body zone) with clear themes than feel obligated to balance out six kinds of borers that seem to have a randomized assortment of abilities.

**Comment by Shadowmech88** [comment]
Part of the reason why headborers are such jacks-of-all-trades chem-wise is that they were left almost entirely unchanged from what regular borers were before the rework, since I figured there was no reason to change them in any major way since they'd be occupying the head, which default borers already did at the time.
As a result their biggest power is being a chem dispenser, compared to arm and leg borers which get more "active" abilities that require more involvement and attention from the borer.
The chest does seem to have gotten the short end of the stick, kind of becoming a worse version of the head.

The idea behind chest borers getting advanced scanning, but not taste blood, was so that they could collaborate with other borers like the head borer, who can keep chem amounts in check via *its* taste blood, or an arm borer, who could fix a broken arm when the chest determines it's broken.

I'm not opposed to reworking them, but that was the idea behind the choices at the time.

**Comment by Shadowmech88** [comment]
Also head borers were intended to have X-ray since even before my rework, when N3X15 reworked them.
https://github.com/d3athrow/vgstation13/commit/568114ab262b47bb1acb8a66c3cb9ecab5f3b66b#diff-c987a2715373b8f320b9a1bd95f692adR191

The prerequisite check was just broken until I fixed it in my rework.

**Comment by AutismSpider** [comment]
Would it be over power if the temp res and damage res chest borers get was turn into type of bone suit  or space suit?

**Comment by Shadowmech88** [comment]
Like, combining them all into a single function that reduces burn and brute damage as well as providing heat, cold, and pressure resistance?
Obviously it would have the cost to justify it, similar to the bone cocoon.

**Comment by AutismSpider** [comment]
IT should have same cost as Bone Cocoon/Bone hammer.

**Comment by aceedex** [comment]
When symbiotic borers were introduced, everyone thought it was unbalanced as fuck, but still had glaring weaknesses and so it stuck

now balancing something that you can have five of in your person at any time around this one thing that was thought to be too strong from the get-go is silly

I'd say sure, balance it that way if you remove multiborers

**Comment by Ephx1** [comment]
What if we change it so that you have to breed borers to get certain chems or genetic alterations and what part of the body it can get into. Then some way for older borers to "update" and get the new abilities the younger borers have, to promote them actually laying eggs and getting people back into the round?

**Comment by nuklearcellphoneg** [comment]
Hi! Please remove borers or balance them, thanks
